### ISSUE TITLE
multi-stage: Cannot read property 'code' of null with relative path install

### ISSUE THREAD

**Issue by aredridel** [npm/npm#7117]
```
:; npm --loglevel=debug install ../dust-makara-helpers/ ../express-bcp47/
npm WARN invalid config loglevel="debug"
▄ ╢░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╟
^Cariastewart@LM-BOS-00872654:womp (new-content-and-react:e8c8a46 Δ )
:; npm --loglevel=silly install ../dust-makara-helpers/ ../express-bcp47/
npm info it worked if it ends with ok
npm verb cli [ 'node',
npm verb cli   '/usr/local/bin/npm',
npm verb cli   '--loglevel=silly',
npm verb cli   'install',
npm verb cli   '../dust-makara-helpers/',
npm verb cli   '../express-bcp47/' ]
npm info using npm@2.2.0
npm info using node@v0.10.33
npm verb node symlink /usr/local/bin/node
npm verb lock using /Users/ariastewart/.npm/_locks/staging-575cbd541edde842.lock for /Users/ariastewart/Projects/womp/node_modules/.staging
npm sill fetchPackageMetaData .
npm sill fetchDirectoryPackageData /Users/ariastewart/Projects/womp
npm sill cache add args [ '@paypal/womp', '/Users/ariastewart/Projects/womp' ]
npm verb cache add spec @paypal/womp@/Users/ariastewart/Projects/womp
npm sill cache add parsed spec { raw: '@paypal/womp@/Users/ariastewart/Projects/womp',
npm sill cache add   scope: '@paypal',
npm sill cache add   name: '@paypal/womp',
npm sill cache add   rawSpec: '/Users/ariastewart/Projects/womp',
npm sill cache add   spec: '/Users/ariastewart/Projects/womp',
npm sill cache add   type: 'directory' }
npm verb addLocalDirectory /Users/ariastewart/.npm/@paypal/womp/0.1.0/package.tgz not in flight; packing
npm verb tar pack [ '/Users/ariastewart/.npm/@paypal/womp/0.1.0/package.tgz',
npm verb tar pack   '/Users/ariastewart/Projects/womp' ]
npm verb tarball /Users/ariastewart/.npm/@paypal/womp/0.1.0/package.tgz
npm verb folder /Users/ariastewart/Projects/womp
npm info prepublish @paypal/womp@0.1.0
npm verb addLocalTarball adding from inside cache /Users/ariastewart/.npm/@paypal/womp/0.1.0/package.tgz
npm sill cache afterAdd @paypal/womp@0.1.0
npm verb afterAdd /Users/ariastewart/.npm/@paypal/womp/0.1.0/package/package.json not in flight; writing
npm verb afterAdd /Users/ariastewart/.npm/@paypal/womp/0.1.0/package/package.json written
npm verb type non_object_property_load
npm verb stack TypeError: Cannot read property 'code' of null
npm verb stack     at /usr/local/lib/node_modules/npm/lib/install.js:160:11
npm verb stack     at /usr/local/lib/node_modules/npm/lib/fetch-package-metadata.js:39:9
npm verb stack     at Parse.<anonymous> (/usr/local/lib/node_modules/npm/lib/fetch-package-metadata.js:114:31)
npm verb stack     at Parse.emit (events.js:92:17)
npm verb stack     at Parse._streamEnd (/usr/local/lib/node_modules/npm/node_modules/tar/lib/parse.js:65:6)
npm verb stack     at BlockStream.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/tar/lib/parse.js:51:8)
npm verb stack     at BlockStream.emit (events.js:92:17)
npm verb stack     at BlockStream._emitChunk (/usr/local/lib/node_modules/npm/node_modules/block-stream/block-stream.js:203:10)
npm verb stack     at BlockStream.flush (/usr/local/lib/node_modules/npm/node_modules/block-stream/block-stream.js:70:8)
npm verb stack     at BlockStream.end (/usr/local/lib/node_modules/npm/node_modules/block-stream/block-stream.js:66:8)
npm verb cwd /Users/ariastewart/Projects/womp
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "--loglevel=silly" "install" "../dust-makara-helpers/" "../express-bcp47/"
npm ERR! node v0.10.33
npm ERR! npm  v2.2.0

npm ERR! Cannot read property 'code' of null
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>
npm verb exit [ 1, true ]

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ariastewart/Projects/womp/npm-debug.log
```

**Comment by aredridel** [comment]
Relative paths are a red herring. Happens with `npm install -g npm@next` too.

**Comment by iarna** [comment]
Fixed in c7296ef
